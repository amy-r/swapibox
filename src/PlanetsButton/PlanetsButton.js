import React from 'react';
import propTypes from 'prop-types';

const PlanetsButton = (props) => {
  return (
    <button onClick= {props.onClick} className= 'PlanetsButton'>Planets</button>
  )
  
}

export default PlanetsButton;