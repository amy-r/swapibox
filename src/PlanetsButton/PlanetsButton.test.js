 /* eslint-disable */ 
import React from 'react';
import ReactDOM from 'react-dom';
import PlanetsButton from './PlanetsButton';
import { shallow, mount } from 'enzyme';
import renderer from 'react-test-renderer';

describe( 'Planets Button', () => {

  it('exists', () => {
  })

  it('renders without crashing', () => {
    const renderedComponent = shallow(<PlanetsButton />, {disableLifecycleMethods: true})
    expect(renderedComponent).toMatchSnapshot();
  })

});