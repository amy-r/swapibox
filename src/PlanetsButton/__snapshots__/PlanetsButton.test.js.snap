// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Planets Button renders without crashing 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <PlanetsButton />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": "PLANETS",
      "className": "PlanetsButton",
      "onClick": undefined,
    },
    "ref": null,
    "rendered": "PLANETS",
    "type": "button",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "PLANETS",
        "className": "PlanetsButton",
        "onClick": undefined,
      },
      "ref": null,
      "rendered": "PLANETS",
      "type": "button",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
